src/connector/src/parser/protobuf/parser.rs

```diff
@@ -250,30 +291,117 @@ pub fn from_protobuf_value(field_desc: &FieldDescriptor, value: &Value) -> Resul
             ScalarImpl::Utf8(enum_symbol.name().into())
         }
         Value::Message(dyn_msg) => {
-            let mut rw_values = Vec::with_capacity(dyn_msg.descriptor().fields().len());
-            // fields is a btree map in descriptor
-            // so it's order is the same as datatype
-            for field_desc in dyn_msg.descriptor().fields() {
-                // missing field
-                if !dyn_msg.has_field(&field_desc)
-                    && field_desc.cardinality() == Cardinality::Required
-                {
-                    let err_msg = format!(
-                        "protobuf parse error.missing required field {:?}",
-                        field_desc
-                    );
-                    return Err(RwError::from(ProtocolError(err_msg)));
+            let flag;
+            if let Some(&DataType::Jsonb) = type_expected {
+                flag = true;
+            } else {
+                flag = false;
+            }
+
+            if dyn_msg.has_field_by_name("type_url") && dyn_msg.has_field_by_name("value") && flag {
+                // The message is of type `Any`
+                let (type_url, payload) = extract_any_info(dyn_msg);
+
+                let payload_field_desc = dyn_msg.descriptor().get_field_by_name("value").unwrap();
+
+                let Some(ScalarImpl::Bytea(payload)) = from_protobuf_value(
+                    &payload_field_desc,
+                    &payload,
+                    descriptor_pool,
+                    type_expected,
+                )?
+                else {
+                    panic!("Expected ScalarImpl::Bytea for payload");
+                };
```

Comment:
I'm not sure if this is necessary at present 🤔️? (And it's a bit confused to do the transformation though 🤣)
The current workaround is to turning the result to a `Vec<String>`,  but I don't know if this is expected.
Since the returned value from `chunk_to_json` is `Result<Vec<String>>`, I'm assuming the ingest value will not be further propagated to downstream, thus turning to a Vec<String> is reasonable.
Also now we support nested message including `ScalarImpl::Struct` and other user defined message, as long as the types inside the message are primitive types.
See the `any-schema.proto` for detailed message examples. 
<img width="640" alt="CleanShot 2023-09-27 at 18 44 49@2x" src="https://github.com/risingwavelabs/risingwave/assets/91381303/51f1cdde-f960-4595-ac86-0d9953ac8d9f">